### PR TITLE
Do not include secondary master by default in LocalAlluxioCluster

### PR DESCRIPTION
**Pull request by bradyoo**
The goal of this PR is to make tests faster/less flaky.

### CODE REVIEW

**Comment by gpang** [comment]
I'm assuming this can make tests faster, and/or less flaky? Could you update the PR description?

File: tests/src/test/java/alluxio/testutils/LocalAlluxioClusterResource.java [hunk 1]
```diff
@@ -105,9 +108,10 @@
    * @param numWorkers the number of Alluxio workers to launch
    * @param configuration configuration for configuring the cluster
    */
-  private LocalAlluxioClusterResource(boolean startCluster, int numWorkers,
-      Map<PropertyKey, String> configuration) {
+  private LocalAlluxioClusterResource(boolean startCluster, boolean includeSecondary,
+                                      int numWorkers, Map<PropertyKey, String> configuration) {
```
Comment (LuQQiu) [inline tests/src/test/java/alluxio/testutils/LocalAlluxioClusterResource.java:112]: Indentation

**Review by LuQQiu — APPROVED** [review]
Left a minor comment. Thanks for making these changes. I agree that secondary master should be included by default in LocalAlluxioCluster because I don't think many people are using it, especially in production environment. 
Even in our local `bin/alluxio-start.sh all SudoMount` we don't launch it.

**Comment by bradyoo** [comment]
Lu, your sentence doesn't make much sense.

**Comment by bradyoo** [comment]
alluxio-bot, merge this please.

**Pull request merged**